src/components/Forums.js

```diff
@@ -0,0 +1,116 @@
+import React, { useRef, useState, useEffect } from "react"
+import { useAuth } from "../contexts/AuthContext"
+import { Link, useHistory } from "react-router-dom"
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import NavigationBar from './NavigationBar';
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+const Forums = () => {
+
+    const { currentUser } = useAuth();
+    const history = useHistory();
+    const [email, setEmail] = React.useState('');
+    const [userID, setUserID] = React.useState('');
+
+    const [forums, setForums] = React.useState([]);
+
+    React.useEffect(() => {
+        setEmail(currentUser.email);
+        loaduserSearchByEmail(currentUser.email);
+        //loadGetForums(); 
+    }, []);
+
+    const loaduserSearchByEmail = (email) => {
+        callApiGetuserSearchByEmail(email)
+            .then(res => {
+                var parsed = JSON.parse(res.express);
+                //console.log(parsed[0].id);
+                setUserID(parsed[0].id);
+            });
+    }
+
+    const callApiGetuserSearchByEmail = async (email) => {
+        const url = `${REACT_APP_API_ENDPOINT}/userSearchByEmail`;
+        //console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                email: email
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    useEffect(() => {
+        loadGetForums();
+    }, []);
+
+    const loadGetForums = async () => {
+        try {
+            const res = await callApiGetForums();
+            const parsed = JSON.parse(res.express);
+            //console.log(parsed[0].forumTitle);
+            setForums(parsed);
+        } catch (error) {
+            console.error(error);
+        }
+    }
+
+    const callApiGetForums = async () => {
+
+        const url = `${REACT_APP_API_ENDPOINT}/getForums`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "GET",
+            headers: {
+                "Content-Type": "application/json",
+            }
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    return (
+        <div id="body">
+
+            <NavigationBar></NavigationBar>
+
+            <Typography
+                variant="h4"
+                gutterBottom
+                component="div">
+                Forums
+            </Typography>
+
+            {forums.map((event) => (
```

Comment:
Maybe we could add some space in between each forum using this: <Box sx={{ m: 2 }} /> after the creation of the first forum? 

Unless we are staying consistent with Events!